src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor() {
```

Comment:
The constructor of the Herbivore class should have a 'name' parameter and pass it to the 'super()' call, like this: 'super(name);'